demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -838,265 +857,170 @@ def validate_no_missing_release_notes(self, modified_files, old_format_files, ad
+                if file_path not in self.ignored_files:
+                    self.ignored_files.add(file_path)
 
-                # ignore changes in doc_files directory
-                elif DOC_FILES_DIR in file_path:
-                    if file_path not in self.ignored_files:
-                        self.ignored_files.add(file_path)
-                        if print_ignored_files:
-                            click.secho('Ignoring file path: {} - doc files'.format(file_path), fg="yellow")
-                    continue
+        return filtered_set, old_format_files
 
-                # identify deleted files
-                if file_status.lower() == 'd' and not file_path.startswith('.'):
-                    deleted_files.add(file_path)
+    def format_file_path(self, file_path, old_path, old_format_files):
+        """Determines if a file is relevant for validation and create any modification to the file_path if needed"""
+        file_type = find_type(file_path)
 
-                # ignore directories
-                elif not os.path.isfile(file_path):
-                    if print_ignored_files:
-                        click.secho('Ignoring file path: {} - directory'.format(file_path), fg="yellow")
-                    continue
+        # ignore unrecognized file types, pack metadata, doc data and test_data
+        if not file_type or file_type == FileType.PACK_METADATA or any(test_dir in str(file_path) for
+                                                                       test_dir in TESTS_AND_DOC_DIRECTORIES):
+            self.ignored_files.add(file_path)
+            return None
 
-                # changes in old scripts and integrations - unified python scripts/integrations
-                elif file_status.lower() in ['m', 'a', 'r'] and find_type(file_path) in [FileType.INTEGRATION,
-                                                                                         FileType.SCRIPT] and \
-                        self._is_py_script_or_integration(file_path):
-                    old_format_files.add(file_path)
-                # identify modified files
-                elif file_status.lower() == 'm' and find_type(file_path) and not file_path.startswith('.'):
-                    modified_files_list.add(file_path)
-                # identify added files
-                elif file_status.lower() == 'a' and find_type(file_path) and not file_path.startswith('.'):
-                    added_files_list.add(file_path)
-                # identify renamed files
-                elif file_status.lower().startswith('r') and find_type(file_path):
-                    # if a code file changed, take the associated yml file.
-                    if find_type(file_data[2]) in [FileType.POWERSHELL_FILE, FileType.PYTHON_FILE]:
-                        modified_files_list.add(file_path)
+        # redirect non-test code files to the associated yml file
+        if file_type in [FileType.PYTHON_FILE, FileType.POWERSHELL_FILE]:
```

Comment:
add JS as well (for rest of the code):
![image](https://user-images.githubusercontent.com/30797606/107623005-5e9e6b00-6c61-11eb-8e31-566bd272fe2d.png)

Reply:
adding